src/doc/guide.md

```diff
@@ -189,42 +180,42 @@ Next up is this line:
 
 This line does all of the work in our little program. There are a number of
 details that are important here. The first is that it's indented with four
-spaces, not tabs. Please configure your editor of choice to insert four spaces
-with the tab key. We provide some [sample configurations for various
-editors](https://github.com/rust-lang/rust/tree/master/src/etc).
-
-The second point is the `println!()` part. This is calling a Rust **macro**,
-which is how metaprogramming is done in Rust. If it were a function instead, it
-would look like this: `println()`. For our purposes, we don't need to worry
-about this difference. Just know that sometimes, you'll see a `!`, and that
-means that you're calling a macro instead of a normal function. Rust implements
-`println!` as a macro rather than a function for good reasons, but that's a
-very advanced topic. You'll learn more when we talk about macros later. One
-last thing to mention: Rust's macros are significantly different from C macros,
-if you've used those. Don't be scared of using macros. We'll get to the details
-eventually, you'll just have to trust us for now.
+spaces (not tabs). Rust's designers and community have settled on spaces as the
+preferred method of indenting code. We provide some
+[sample configurations for various editors](https://github.com/rust-lang/rust/tree/master/src/etc).
+
+The second point is the `println!()` part. This calls a Rust **macro**, which is
+how metaprogramming is done in Rust. If it were a function instead, it would
+look like this: `println()`. For our purposes, we don't need to worry about this
+difference. Just know that sometimes, you'll see a `!`, and that means you're
+calling a macro instead of a normal function. Rust implements `println!` as a
+macro rather than a function for good reasons, but that's an advanced
+topic. You'll learn more when we talk about macros later. One last thing to
+mention: Rust's macros are significantly different than C macros, if you've used
+those. Don't be scared of using macros. We'll get to the details eventually, but
+you'll just have to trust us for now.
 
 Next, `"Hello, world!"` is a **string**. Strings are a surprisingly complicated
 topic in a systems programming language, and this is a **statically allocated**
 string. We will talk more about different kinds of allocation later. We pass
-this string as an argument to `println!`, which prints the string to the
-screen. Easy enough!
+this string as an argument to `println!`, which prints the string and a linefeed
+character to the screen. Easy enough!
```

Comment:
"prints the string and a newline to the screen."

Newline is more correct and platform independent.